kolibri/content/content_db_router.py

```diff
@@ -80,7 +80,7 @@ def get_content_database_connection(alias=None):
             raise KeyError("Content DB '%s' is empty!!" % alias)
 
     # if possible, attach the default database to the content database connection to enable joins
-    _attach_default_database(alias)
+    # _attach_default_database(alias) # Fix 1818.1 db locked and cannot be removed on Windows
```

Comment:
How does attaching the default database make a difference to the behaviour of the content DB?